README.md

```diff
@@ -1,54 +1,84 @@
-# Project Name
+# MOVIE WEBAPP USING AN API
 
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+> The JavaScript capstone project is about building your own web application based on an external API. We selected an API(TvMaze) that provides data about movies that we like and then built the webapp around it. The webapp will has 2 user interfaces: A home page showing a list of items that you can "like."
+A popup window with more data about an item that you can use to comment on it or reserve it for a period of time.
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
+## Video Presentation
+[Video Presentation](https://drive.google.com/file/d/1yn0rZc0MxNsY-SoR4WWvCRbwZ2jj5DmE/view?usp=sharing)
```

Comment:
- [ ] while you tried to make a video presentation for your project, you forget to highlight some codes you're proud of. Also, make sure that all teammate participate while explaining the core functionality of your web application. Kindly, make another video for 5 minutes minimum where you present your project as required.